src/javascript/app/pages/cashier/payment_agent_list.js

```diff
@@ -67,6 +67,22 @@ const PaymentAgentList = (() => {
                 });
             }
 
+            let urls = '';
+            if (agent.urls && agent.urls.length > 0) {
+                agent.urls.map((item, index) => {
+                    urls += index ? ',' : '';
+                    urls += `<a href="${item.url}" target='_blank'>${item.url}</a>`;
+                });
+            }
+
+            let phone_numbers = '';
+            if (agent.phone_numbers && agent.phone_numbers.length > 0) {
```

Comment:
```suggestion
            if (agent.phone_numbers?.length > 0) {
```

Reply:
optional conditioning is not supported in binary